parameters/src/mainnet/mod.rs

```diff
@@ -29,9 +29,6 @@ const REMOTE_URL: &str = "https://parameters.aleo.org/mainnet";
 impl_local!(Degree15, "resources/", "powers-of-beta-15", "usrs");
 #[cfg(feature = "wasm")]
 impl_remote!(Degree15, REMOTE_URL, "resources/", "powers-of-beta-15", "usrs");
-#[cfg(not(feature = "wasm"))]
-impl_local!(Degree16, "resources/", "powers-of-beta-16", "usrs");
```

Comment:
Why remove degree 16 but not degree 15? Or the shifted degrees?

Also CC @iamalwaysuncomfortable because he may still have context on the WASM stuff

Reply:
Removal due to `crates.io` restrictions on size (10MB maximum). Degree 16 now defaults to `impl_remote`.

Should not affect wasm at all, as it was already using `impl_remote`.